content/en/docs/architecture/branching.md

```diff
@@ -92,4 +80,24 @@ Yes. When final freeze happens for release X.Y, the release branch for that vers
 ## Do I need to make sure branched CI Operator configs are up to date for release branches?
 No. When the branch is cut and goes live, the current CI Operator configuration for the development branch is used to seed the configuration for the release branch. These are not DRY intentionally to allow for drift between CI for branches.
 
+## What was the No-Feature-Freeze Process?  I hear my team still uses it.
+
+As the name suggests No-Feature-Freeze refers to a time when we used to have a
+Feature Freeze.  During the legacy feature freeze we had not yet branched to
+allow for development to start on the N+1 minor release but we enforced all PRs
+to reference bug fixes.
+
+A few years ago a few teams participated in a pilot process that allowed them
+to continue developing past the Feature Freeze phase.  Repositories that belong
+to these teams merge pull requests to their `master/main` branches if they are
+labelled with one of the following sets of labels:
+
+- `lgtm`, `approved`, `bugzilla/valid-bug` (for bugfix pull requests)
+- `lgtm`, `approved`, `px-approved`, `docs-approved`, `qe-approved` (for feature pull requests)
+
+The thing to know is that we don't offer this to new repositories.  These days
```

Comment:
Can we say something like:

> These days we don't have a Feature Freeze as in years past.  Some teams may use these labels to encourage pre-merge review of features, based on their own assessment of that pre-merge review by PX, docs, and QE generates a faster overall velocity.  This assessment is expected to be a per-team call, depending on the quality or pre-merge review and the usual turnaround time needed to deliver it.

Because the cluster-version operator jumped into these labels in #45221 to get more convenient pre-merge review, not to get No-Feature-Freeze handling.  And I like pre-merge review,  and don't see why we'd want to discourage other components from taking a similar approach now.

Reply:
I like that.  Opening https://github.com/openshift/ci-docs/pull/489